experimental/packages/opentelemetry-exporter-prometheus/src/PrometheusSerializer.ts

```diff
@@ -184,15 +184,15 @@ export class PrometheusSerializer {
     return str;
   }
 
-  serializeScopeMetrics(scopeMetrics: ScopeMetrics) {
+  private serializeScopeMetrics(scopeMetrics: ScopeMetrics) {
```

Comment:
A private method needs to be prefixed with a `_` so that even in JS the method is being observed as somewhat "private".

```suggestion
  private _serializeScopeMetrics(scopeMetrics: ScopeMetrics) {
```

Reply:
👍🏻 I'll do that for the rest of them too.